sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -584,7 +653,27 @@ export class NotificationHubsServiceClient {
 }
 
 // @public
-export interface NotificationOutcomeCollectionItem {
+export interface NotificationHubsClientOptions extends CommonClientOptions {
+}
+
+// @public
+export interface NotificationHubsMessageResponse extends NotificationHubsResponse {
+    failure: number;
```

Comment:
nit `failureCount`, `failedMessageCount`, or somesuch might make this more intuitive. The names `failure` and `success` feel boolean to me